### ISSUE TITLE
Watch Extension

### ISSUE THREAD

**Issue by noman-ahmad** [pafry7/wideget-expo-example#1]
Do you think that it would be possible to extend this kind of plugin paradigm to also support apple watch/android watch extensions?

**Comment by pafry7** [comment]
Yeah, I think it's possible and it would work similarly to the widget. You will have to:
1) Copy watch extension files during prebuild
2) Add target to the project via config plugin
3) Write your own module handling communication between the app and the extension or use existing solutions:
https://github.com/mtford90/react-native-watch-connectivity and https://github.com/circles-00/react-native-wear-communication-module
